control/margins.py

```diff
@@ -296,11 +296,10 @@ def phase_crossover_frequencies(sys):
     (array([ 1.73205081,  0.        ]), array([-0.5 ,  0.25]))
     """
 
+    if not sys.issiso(): 
+        raise ValueError("MIMO systems not yet implemented.")
```

Comment:
```suggestion
        raise NotImplementedError("MIMO systems not yet implemented.")
```

Reply:
Looks like there is actually a ``ControlMIMONotImplemented`` error. will use that one in next commit.